bundles.rst

```diff
@@ -74,7 +80,7 @@ of the bundle. Now that you've created the bundle, enable it::
     // config/bundles.php
     return [
         // ...
-        App\Acme\TestBundle\AcmeTestBundle::class => ['all' => true],
```

Comment:
To be backported in 4.4?

Reply:
Yup, the whole bundle doc should be updated since 4.4.

Reply:
I will create a separate PR for that.

Reply:
I don't think backporting is needed. We often don't do big rewrites on old versions. We simply don't have much resources on documentation, so getting current in good shape is hard enough already.
Besides that, nobody should start writing a new bundle in their Symfony 4.4 application and the current 4.4 doc already served 4.4 users for 2.5 years, we'll manage that the final 6 months as well :)